.github/workflows/go.yml

```diff
@@ -23,5 +23,5 @@ jobs:
         run: go build
         working-directory: ${{ env.relativePath }}
       - name: Test SDK
-        run: go test
+        run: go test ./...
```

Comment:
not sure what ... means? did you mean ../..?

Reply:
aaah thanks a lot for that distinction. I feel like we probably have a similar issue in graph core and in the http package as well. Would you mind fixing that for those too please?

Reply:
Will do, fixing them on the backing store pr as well.